webpack.config.js

```diff
@@ -124,5 +127,47 @@ module.exports = [
         output: Object.assign({}, config.output, {
             library: 'JitsiMeetExternalAPI'
         })
-    })
+    }),
+
+    // React application
```

Comment:
We don't want a new entry point (respectively, Webpack configuration). We want to modify the implementation of the existing entry point `app.bundle` without changing its export i.e. it should remain `APP` and the jitsi-meet application should continue to work (as seen from the outside) after merging your PR without loss of functionality, as if we haven't introduced React. I've merged the HasteResolverPlugin and the babel presets into the existing entry point `app.bundle` so you'll have to make sure that (a minimal set of) the remaining modifications are incorporated there as well.